src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -51,7 +51,7 @@ public sealed class UIActionEngine
         /// <summary>
         /// Perform an install or uninstall user action.
         /// </summary>
-        /// <remarks>This needs to be called from a background thread. It may hang on the UI thread.</remarks>
+        /// <remarks>This needs to be called from a background thread. It may make the UI thread stop responding.</remarks>
```

Comment:
use freeze like in other places where you've replaced hang?

Reply:
@nkolev92 the guidance is to replace "hang" with "stop responding" when referring to UI or threads. The freeze concept that he used in other place was when it didn't make sense to use "stop responding" such as in the test code where the parameter is "-hang" and "-stopresponding" just doesn't make a lot of sense.

Reply:
I will follow with current guidance to keep consistency.

Reply:
Can you link that guidance? 

I followed https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1322667 but the links there don't take me to any particula recommendations, I'm sure I'm missing something.

Reply:
I found it through [an e-mail](https://policheck.azurewebsites.net/Pages/DisplayTermDetails.aspx), anyone has any idea how to get to that through the issues?